start_listener.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2014 Cloudbase Solutions Srl
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from argus.debug import listener
+from argus.util import get_config as config
+
+Listener = listener.BasicListener(config().debug)
```

Comment:
@alexcoman There is a reason for which I included the start_listener.py in the main folder. And that is because you can't include it as a module. For debug mode to actually work you have to start this first python start_listener.py , then in another console run your tests.